source/scripts/pages/EventPage/index.js

```diff
@@ -280,6 +271,7 @@ const EventPage = React.createClass({
 								},
 							]}
 						/>
+						{!isFetchingEvent && this.renderDataUpdatedSnackbar()}
```

Comment:
Please make sure that this is a correct business logic behaviour.

Reply:
Yep, it works nice!